pom.xml

```diff
@@ -328,14 +331,10 @@
                             <runOrder>hourly</runOrder>
                             <argLine>
                                 -Duser.timezone=UTC
-                                --add-exports java.base/jdk.internal.ref=ALL-UNNAMED
-                                --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
-                                --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
-                                --add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED
-                                --add-opens=jdk.compiler/com.sun.tools.javac=ALL-UNNAMED
-                                --add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED
-                                --add-opens=java.base/java.lang=ALL-UNNAMED
-                                --add-opens=java.base/java.lang.reflect=ALL-UNNAMED
+                                --add-exports java.base/jdk.internal.ref=ALL-UNNAMED --add-exports=java.base/sun.nio.ch=ALL-UNNAMED
+                                --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED
+                                --add-opens=jdk.compiler/com.sun.tools.javac=ALL-UNNAMED --add-exports=jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED
+                                --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED
                                 --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED
```

Comment:
I bet we could achive this without having to duplicate all this stuff by using https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#systemPropertyVariables

Reply:
These are not system properties but command line args to `java`. But perhaps I misunderstood your comment?

Reply:
They're all added in the root parent pom. The only reason we duplicate them here is to add the -Duser.timezone=UTC parameter. If we use the system property configuration we don't need to duplicate the argLine as it should be inherited?